src/i18n/en_AU.ts

```diff
@@ -348,7 +348,7 @@ export default {
             possibleBuildings: [6, 19],
             schooling: {
                 Police: {
-                    'Police Motorcycle': {
+                    'Motor Officer': {
```

Comment:
this is also wrong, here it must be exactly the same as in the other picture or in this picture.
https://cdn.discordapp.com/attachments/544869019673559060/918244105778253914/unknown.png